w3d_translator/w3d_translator.py

```diff
@@ -55,40 +52,37 @@ def farewell_error():
 
 
 # Translate a single project
-def translate_project(name: str, project_dir: str, out_dir: str):
-    # Create Unity project and copy original files
-    # unity_dir = os.path.join(out_dir, name)
-    # try:
-    # create_project(unity_dir)
-    # copy_files(project_dir, unity_dir)
-    # add_empty_scene(unity_dir)
-    # except UnityError as e:
-    # handleError(e)
-
-    # Translate xml files in individual threads
-    xml_files = [
-        os.path.join(project_dir, file)
-        for file in os.listdir(project_dir)
-        if file.endswith(".xml")
-    ]
-    for file in xml_files:
-        typer.echo(f"Translating file: {os.path.basename(file)}")
-
-        # Will return a yaml file
-        xml_to_unity(file)
-
-
-def handleError(e):
-    typer.echo(e, err=True)
-    global is_error
-    is_error = True
+def translate_project(project_dir: Path, out_dir: Path):
+    try:
+        typer.echo(f"Translating project:\t {cyan(project_dir.name)}")
+        validate_project(project_dir)
+
+        # ! Don't create a new Unity project while in development
+        # Create Unity project and copy original files
+        # unity_dir = out_dir / project_dir.name
+        #     create_project(unity_dir)
+        #     copy_files(project_dir, unity_dir)
+        #     add_empty_scene(unity_dir)
```

Comment:
might be worth adding some sort of `dev` flag to your command line args so you don't need to comment this out

Reply:
Added! Definitely useful